contracts/ERC20Safe.sol

```diff
@@ -101,6 +133,18 @@ contract ERC20Safe is Initializable, BridgeRole, Pausable {
             require(burnBalance == 0, "Stored tokens must have 0 burn balance!");
             initSupply(token, totalBalance);
         }
+
+        if (singleTxThreshold == 0) {
+            singleTransactionThreshold[token] = defaultSingleTransactionThreshold;
```

Comment:
Question: do you think it will be better that, instead of using default values here, we should error as to prevent as much as we can bad values?
Valid also for aggregateThreshold